ara/ui/templates/index.html

```diff
@@ -146,11 +151,14 @@
             </td>
             <td class="text-center">
               {% if not static_generation %}
-                <a href="{% url 'ui:index' %}?controller={{ playbook.controller }}">{{ playbook.controller }}</a>
+                <a href="{% url 'ui:index' %}?controller={{ playbook.controller }}">{{ playbook.controller  }}</a>
```

Comment:
```suggestion
                <a href="{% url 'ui:index' %}?controller={{ playbook.controller }}">{{ playbook.controller }}</a>
```

Looks like an unintended extra whitespace here

Reply:
Good catch :D Fixed.